frontend/app/components/page-header-brand.tsx

```diff
@@ -1,20 +1,37 @@
+import { Link } from '@remix-run/react';
+
 import { useTranslation } from 'react-i18next';
 
 import { LanguageSwitcher } from '~/components/language-switcher';
 import { getAltLanguage } from '~/utils/locale-utils';
 
-export function PageHeaderBrand() {
+export interface PageHeaderBrandProps {
+  link?: boolean;
+}
+
+export function PageHeaderBrand({ link = false }: PageHeaderBrandProps) {
   const { i18n, t } = useTranslation(['gcweb']);
 
   const altLanguage = getAltLanguage(i18n.language);
   const altLogoContent = <span lang={altLanguage}>{i18n.getFixedT(altLanguage)('gcweb:header.govt-of-canada.text')}</span>;
-
+  const pageHeaderLogo = <img className="h-8 w-auto" src={`/assets/sig-blk-${i18n.language}.svg`} alt={t('gcweb:header.govt-of-canada.text')} property="logo" width="300" height="28" decoding="async" />;
+  const readOnlyLogo = <span className="sr-only">/ {altLogoContent}</span>;
   return (
     <div id="wb-bnr">
       <div className="container flex items-center justify-between gap-6 py-2.5 sm:py-3.5">
         <div property="publisher" typeof="GovernmentOrganization">
-          <img className="h-8 w-auto" src={`/assets/sig-blk-${i18n.language}.svg`} alt={t('gcweb:header.govt-of-canada.text')} property="logo" width="300" height="28" decoding="async" />
-          <span className="sr-only">/ {altLogoContent}</span>
+          {link ? (
+            <Link to="https://canada.ca/" property="url">
```

Comment:
Use normal `a` html tag.

```suggestion
            <a href="https://canada.ca/" property="url">
```